config/locales/fr.yml

```
@@ -160,12 +160,14 @@ fr:
         default_value: Valeur par défaut
         description: Description
       customer:
+        authorize_overtake: Autoriser le retard
         ref: Référence
         ref_help: La référence doit être unique
         name: Nom
         description_title: Description
         description_label: Informations du compte
         description_help: Détails sur le compte et les utilisateurs courants
+        enable_optimization_soft_upper_bound: Authoriser les retards
```

```suggestion
        enable_optimization_soft_upper_bound: Autoriser les retards
```
Please put it with the others enable_ entries